src/MSBuild.Conversion.SDK/SemanticVersion.cs

```diff
@@ -0,0 +1,221 @@
+﻿using System;
+using System.Text.RegularExpressions;
+
+namespace MSBuild.Conversion
+{
+    public class SemanticVersion : IComparable<SemanticVersion>, IEquatable<SemanticVersion>
```

Comment:
I thought there was a type available here that we'd be able to use. At least I know that NuGet had to implement one. Is that not accessible?